core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -44,7 +45,7 @@ class AnalyzerManager(
       case Some(moduleId) =>
         f(req, moduleId)
       case None =>
-        sender ! EnsimeServerError(s"Couldn't find the project for ${fileInfo.file}")
+        sauron forward req
```

Comment:
this is a bit hacky... the `f` doesn't get called if there is no `moduleId`. Perhaps it's time to rethink this borrow pattern?